docs/workflow.rst

```diff
@@ -332,6 +340,9 @@ For example:
     Fabric deployment should be very easy to deploy on both local and remote machine.
     This is the work on local part.
 
+.. note::
+    "@" is used before *<issue-key>* because github, and gitlab have the "#" character. It's fine for the command *git commit -m "#<issue-number>"* but not for *git commit -a* or new commit or *$ git commit --amend* for an existing commit to open on the VIM editor.
```

Comment:
"@" is used before *<issue-key>* because github, and gitlab's issue keys have the "#" character. It's fine for the command *git commit -m "#<issue-number>"* but not for *git commit -a* for a new commit or *$ git commit --amend* for an existing commit to open on the VIM editor. That's the reason why we should use "@" character, it means "address <issue-key>".